os/board/rtl8730e/src/rtl8730e_alc1019.c

```diff
@@ -43,7 +43,11 @@ extern FAR struct i2s_dev_s *amebasmart_i2s_initialize(uint16_t port);
  * Pre-processor Definitions
  ****************************************************************************/
 /* i2c config */
+#if CONFIG_BOARD_REVISION_VERSION == 5
```

Comment:
If you want to keep like this, you should add this definition in every defconfig.

Reply:
I added code to check if there is no this conifg